src/cloneCommand.ts

```diff
@@ -56,7 +56,11 @@ export const gitCloneCommandPlugin: JupyterFrontEndPlugin<void> = {
               gitModel as GitExtension,
               Operation.Clone,
               trans,
-              { path: fileBrowserModel.path, url: result.value }
+              {
+                path: fileBrowserModel.path,
+                url: result.value.url,
+                not_versioning: result.value.not_versioning
```

Comment:
This is a shortcut syntax that tells the compiler to copy all (key, value) from `result.value` into the new object:

```suggestion
                ...result.value
```